<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOSettingsWindow</class>
 <widget class="QWidget" name="IOSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupIO1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>781</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>IO 1 (BNC)</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioIO1FSOut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>15</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Frame Sync Output</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO1TrigIn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>15</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Record End Trigger</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO1TriggeredShutter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exposure Trigger</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO1Invert">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>105</y>
      <width>97</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO1Debounce">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Debounce</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO1WeakPull">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>105</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1mA Pullup</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO1Pull">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>20mA Pullup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO1None">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>15</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO1ShutterGating">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>105</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Shutter Gating</string>
    </property>
   </widget>
   <widget class="CamDoubleSpinBox" name="spinIO1Thresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>136</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>7200.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO1ToggleTrig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Toggle</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupIO2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>781</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>IO 2</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioIO2FSOut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>15</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Frame Sync Output</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO2TrigIn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>15</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Record End Trigger</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO2Invert">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>105</y>
      <width>97</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO2Debounce">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Debounce</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO2Pull">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>20mA Pullup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO2None">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>15</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO2TriggeredShutter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exposure Trigger</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO2ShutterGating">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>105</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Shutter Gating</string>
    </property>
   </widget>
   <widget class="CamDoubleSpinBox" name="spinIO2Thresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>136</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>7200.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdOK">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupIO3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Input 3 (Isolated)</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="chkIO3Invert">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>105</y>
      <width>97</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIO3Debounce">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Debounce</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO3TrigIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Record End Trigger</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO3TriggeredShutter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exposure Trigger</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO3ShutterGating">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Shutter Gating</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIO3None">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>15</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdApply">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CamDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>camdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdCancel</tabstop>
  <tabstop>cmdOK</tabstop>
  <tabstop>cmdApply</tabstop>
  <tabstop>chkIO3Invert</tabstop>
  <tabstop>radioIO3ShutterGating</tabstop>
  <tabstop>chkIO3Debounce</tabstop>
  <tabstop>radioIO3TriggeredShutter</tabstop>
  <tabstop>radioIO3None</tabstop>
  <tabstop>radioIO3TrigIn</tabstop>
  <tabstop>chkIO2Invert</tabstop>
  <tabstop>radioIO2ShutterGating</tabstop>
  <tabstop>chkIO2Pull</tabstop>
  <tabstop>chkIO2Debounce</tabstop>
  <tabstop>radioIO2TriggeredShutter</tabstop>
  <tabstop>radioIO2FSOut</tabstop>
  <tabstop>radioIO2None</tabstop>
  <tabstop>radioIO2TrigIn</tabstop>
  <tabstop>spinIO2Thresh</tabstop>
  <tabstop>chkIO1WeakPull</tabstop>
  <tabstop>chkIO1Invert</tabstop>
  <tabstop>radioIO1ShutterGating</tabstop>
  <tabstop>chkIO1Pull</tabstop>
  <tabstop>chkIO1Debounce</tabstop>
  <tabstop>radioIO1TriggeredShutter</tabstop>
  <tabstop>radioIO1FSOut</tabstop>
  <tabstop>radioIO1None</tabstop>
  <tabstop>radioIO1TrigIn</tabstop>
  <tabstop>spinIO1Thresh</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
